yarn.lock

```diff
@@ -4433,10 +4433,10 @@ soroban-client@0.5.1, soroban-client@^0.5.1:
     stellar-base "8.2.2-soroban.12"
```

Comment:
@esteblock why is `soroban-client@0.5.1` still referenced in the `yarn.lock`? is this a stale remnant or is there an actual issue here?

Reply:
Oh, this must be a mistake, because it's the root yarn.lock

In monorepos, with Lerna, the package/my-package/yarn.lock it's not updated.
But the ./yarn.lock should be updated.
So probably this is an error. I will check

Reply:
This 

> @esteblock why is `soroban-client@0.5.1` still referenced in the `yarn.lock`? is this a stale remnant or is there an actual issue here?

This was because ./utils, ./events and ./wallet-data where using `soroban-client@0.5.1` 
I will upgrade to 0.7.0